### DISCUSSION TITLE
Paste equivalent to Evergreen's TagInput

### DISCUSSION THREAD

**Discussion by mschasz** [twilio-labs/paste#3423] [General] [answered]
Hi team, 

I'm trying to migrate this `TagInput` component that creates a tag in the input component after a users presses enter. I don't see a component that exactly matches this one, so I wanted to know if there's any guidance on how to migrate this design? I was thinking of either using a form pill group above an input component or prompting the user to enter a comma separated input, then generating the tags after the user presses enter or submitting the form. Thanks!

<img width="1587" alt="image" src="https://github.com/twilio-labs/paste/assets/109534751/fcc97f23-eebc-49ca-8a88-532ea2c5534f">

**Comment by loreina** [comment]
Hi @mschasz,

Paste's equivalent would be our [Multiselect Combobox](https://paste.twilio.design/components/multiselect-combobox). You should be able to compose an option within Combobox for users to create their own options. Thanks!

**Comment by SiTaggart** [comment, marked as answer]
If you don't actually have predefined values, and a combobox doesn't make much sense, you can also just compose your own version using a collection of Paste components like you suggest, or as discussed in this previous discussion https://github.com/twilio-labs/paste/discussions/3311
